src/IROperator.h

```diff
@@ -1636,6 +1636,37 @@ inline Expr count_trailing_zeros(Expr x) {
                                 {x}, Internal::Call::PureIntrinsic);
 }
 
+/** Divide two integers, rounding towards zero. This is the typical
+ * behavior of most hardware architectures, which differs from
+ * Halide's division operator, which is Euclidean (rounds towards
+ * -infinity). */
+inline Expr div_round_to_zero(Expr x, Expr y) {
```

Comment:
I was thinking it was best to just handle it in the backend, but I'm now realizing that this means the simplifier will miss things it could have seen. I'll change it.